home/.chezmoi.yaml.tmpl

```diff
@@ -140,6 +140,11 @@ data:
     docker:
       doRegion: nyc1
     domain: "{{ $domain }}"
+    gitlab:
+      glurl: "https://gitlab.com/"
+      runnerDescription: "Docker executor"
```

Comment:
Let's get rid of this and automatically populate the description and include the name of the HOST in it

Reply:
Updated to add the host name in the description field, in addition to the value provided here. The reason I would prefer leaving this as an input is to distinguish the runners when multiple runners are registered on a given host. There could be one per version of a SDK/software/language or there  could be multiple runners each with a separate set of tools. Having a description will allow identifying them easily.

Reply:
ok sure